### ISSUE TITLE
Synonyms and antonyms functionnalities are down

### ISSUE THREAD

**Issue by alexandreraufast** [geekpradd/PyDictionary#30]
```print (dictionary.synonym("..."))
print (dictionary.antonym("..."))``` not working.
Meanings still working

**Comment by kenseii** [comment]
Any plan to fix this or it is open for contributors?

**Comment by geekpradd** [comment]
Yes there is a plan for fixing this. I am open to contributors as well. Fix should be done by end of this month.
Thanks for your patience.

**Comment by geekpradd** [comment]
Fixed as of version 2.0.0
